app/lib/services/voip/MediaCallEvents.ios.test.ts

```
@@ -169,186 +138,120 @@ const activeCallBase = {
 	beforeEach(() => {
 		jest.clearAllMocks();
-		resetMediaCallEventsStateForTesting();
-		Object.keys(mockNativeVoipListeners).forEach(k => delete mockNativeVoipListeners[k]);
-		mockAddEventListener.mockImplementation(() => ({ remove: jest.fn() }));
-		(NativeVoipModule.getInitialEvents as jest.Mock).mockReset();
-		(RNCallKeep.getInitialEvents as jest.Mock).mockReset();
 		getState.mockReturnValue({ setNativeAcceptedCallId: mockSetNativeAcceptedCallId });
 	});
 
-	it('returns true and applies REST signals when CallKit shows answered and host matches workspace', async () => {
+	it('applies REST signals and returns true for iOS cold-start same-workspace', async () => {
```

ESLint

[error] Async arrow function has no 'await' expression.

(require-await)